packages/core/content-releases/admin/src/pages/ReleaseDetailsPage.tsx

```diff
@@ -317,9 +336,7 @@ export const ReleaseDetailsLayout = ({
   }
 
   const totalEntries = release.actions.meta.count || 0;
-  const createdBy = release.createdBy.lastname
-    ? `${release.createdBy.firstname} ${release.createdBy.lastname}`
-    : `${release.createdBy.firstname}`;
+  const hasCreatedByUser = Boolean(getCreatedByUser());
```

Comment:
Do we really need this boolean? Can we just check if the returning value from getCreatedByUser is not a null value?

Reply:
Isn't that what the check is doing? Return false if falsy (ie null) return true if truthy (ie we have a string)